### DISCUSSION TITLE
SqliteDatabase Can Not Open Columns Tab

### DISCUSSION THREAD

**Discussion by HakanDOGAN1981** [dbeaver/dbeaver#14827] [Q&A] [answered]
Hello,

When i try to open my sqlite database, gat an error 
"SQL Error [1]: [SQLITE_ERROR] SQL error or missing database (too many terms in compound SELECT)
 [SQLITE_ERROR] SQL error or missing database (too many terms in compound SELECT)
 [SQLITE_ERROR] SQL error or missing database (too many terms in compound SELECT)".

I don't get this error with other database editors(Navicat 15 for SQLite, SQLiteStudio, SQLite Expert ...).

Can you fix this? [I added my database link.](https://drive.google.com/file/d/1G2B9ZKNM0GA6CmQSKBEvp9kWnNUtvzAK/view?usp=sharing)

Thank you.

**Comment by ShadelessFox** [comment]
Hello @HakanDOGAN1981,

I cannot reproduce your issue:
![image](https://user-images.githubusercontent.com/35821147/145095437-4c185b3b-9dbf-4600-a691-adf59af1f720.png)

Please let me know:
1. What is the version of DBeaver
2. What is the version of SQLite driver

**Reply by HakanDOGAN1981** [reply]
Thanks for your return.

Firstly i could not open "TBL" table. (I am sorry. I forgat it...)

1. Version 21.3.0.202111281534

2. ![image](https://user-images.githubusercontent.com/34808406/145108846-e7c3ce2b-fd06-483b-a299-cdf47a645212.png)
[VeriTabani.zip](https://github.com/dbeaver/dbeaver/files/7671719/VeriTabani.zip)

**Comment by ShadelessFox** [comment, marked as answer]
Setting driver property `limit_compound_select` to `0` should help:

![java_7HwUbhtCKB](https://user-images.githubusercontent.com/35821147/145123591-faa9314b-f87a-4cee-bae9-0be9b3f3b6a8.png)

**Reply by HakanDOGAN1981** [reply]
Thanks, it is work. Please this setting to default.
